cli/commands/login.go

```diff
@@ -76,6 +81,17 @@ func (cmd *Login) Run(scope scope.Scope, c *cli.Context) {
 		cmd.network.BrooklynUrl = cmd.network.BrooklynUrl[0 : len(cmd.network.BrooklynUrl)-1]
 	}
 
+	// Prompt for username if not supplied
+	if cmd.network.BrooklynUser == "" {
+		reader := bufio.NewReader(os.Stdin)
+		fmt.Print("Enter Username: ")
+		user, err := reader.ReadString('\n')
+		if err != nil {
+			error_handler.ErrorExit(err)
+		}
+		cmd.network.BrooklynUser = strings.TrimSpace(user)
```

Comment:
You can end up with an empty string for the user, should pull this out into a function and potentially retry if no name is supplied.